cli/lsp/urls.rs

```diff
@@ -96,7 +87,11 @@ impl LspUrlMap {
         specifier.clone()
       } else {
         let specifier_str = if specifier.scheme() == "data" {
-          let media_type = data_url_media_type(specifier);
+          let data_url = DataUrl::process(specifier.as_str())
+            .map_err(|e| type_error(format!("{:?}", e)))?;
```

Comment:
Any suggestions on what error type would be appropriate here?

Reply:
`URIError`

Reply:
Thanks! :)